content/about/main/index.md

```diff
@@ -7,29 +7,29 @@ number_categories: 3 # set to zero to exclude
   
-  These communities also gave me the opportunity to improve myself, in 2019 I become an [RStudio Certified Trainer](/blog/2020_rstudiocertification_en/), since 2020 I'm an [Software Carpentries Certified Instructor](https://carpentries.org/instructors/) and since 2021 I'm a  [Carpentries Certified Trainer](https://carpentries.org/trainers/). In 2022 the community elected me as a member of the [The Carpentries Execute Council](https://carpentries.org/blog/2021/12/announcing-2022-executive-council/). 
+  These communities also gave me the opportunity to improve myself, in 2019 I became an [RStudio Certified Trainer](/blog/2020_rstudiocertification_en/), since 2020 I'm an [Software Carpentries Certified Instructor](https://carpentries.org/instructors/) and since 2021 I'm a  [Carpentries Certified Trainer](https://carpentries.org/trainers/). In 2022 the community elected me as a member of the [The Carpentries Execute Council](https://carpentries.org/blog/2021/12/announcing-2022-executive-council/). 
   
-  I delivery a lot of talk at events and courses and thanks to these communities I become an international bilingual speaker and teacher.  
+  I am a prolific international bilingual speaker and teacher thanks to these communities.  
   
-  The community experience is so empowering that encouraged me to co-found in 2020 a new community called [MetaDocencia](https://metadocencia.netlify.app/), an open, free, volunteer-lead, not-for-profit, educational organization that teach how to teach and technical skill to people from undeserved countries.
+  The community experience is so empowering that it encouraged me to co-found, in 2020, a new community called [MetaDocencia](https://metadocencia.netlify.app/), an open, free, volunteer-lead, not-for-profit, educational organization that teach how to teach and technical skill to people from undeserved countries.
   
-  In 2019 I started to teach at the undergraduate and graduate level for two diploma courses (Data Science in Public Policy and New Technologies for Agriculture) and a Bachelor's Degree in Data Science.  I also helped to create a master's degree in Data Science.
+  In 2004 I met the man who became my partner in life and in 2012, 2013 and 2015 we became proud parents. In 2012 we lost Ana, our first child. Now we enjoy watching with wonder what beautiful people our other two kids are becoming.
```

Comment:
Maybe this paragraph can start with a phrase indicating it's about your personal life, but it does not have to. Also, I can't find an example of such a phrase so that's not a very useful comment. :see_no_evil: